README.md

```diff
@@ -39,75 +37,6 @@ Each package has its own **node_modules** directory that contains symbolic links
-6. Check for API signature changes: `rush extract-api`. This will update the signature files, located in `common/api`. **Note:** before doing this, first do the following:
-    - Be sure that your branch is up to date with the target branch (i.e. `git merge origin/master`)
-    - Cleanup your build output: `rush clean`
-    - Rebuild the project: `rush build`
-7. Review any diffs to the API signature files in the `common/api` directory to ensure they are compatible with the intended release of the package.
-    - If any differences are in packages not modified on this branch, revert the changes before committing.
-8. Add changelog entry (which could potentially cover several commits): `rush change`
-9. Follow prompts to enter a change description or press ENTER if the change does not warrant a changelog entry. If multiple packages have changed, multiple sets of prompts will be presented. If the changes are only to non-published packages (like **display-test-app**), then `rush change` will indicate that a changelog entry is not needed.
-10. Completing the `rush change` prompts will cause new changelog entry JSON files to be created.
-11. Add and commit the changelog JSON files and any API signature updates.
-12. Publish changes on the branch and open a pull request.
-
-> If executing scripts from `package.json` files in any of the subdirectories, we recommend using [`rushx`](https://rushjs.io/pages/commands/rushx/) over `npm`.
-
-If using the command line, steps 8 through 11 above can be completed in one step by running `rushchange.bat` from the imodeljs root directory.
-Only use `rushchange.bat` if none of the changes require a changelog entry.
-> Note: The CI build will break if changes are pushed without running `rush change` and `rush extract-api` (if the API was changed). The fix will be to complete steps 6 through 11.
-
-Here is a sample [changelog](https://github.com/microsoft/rushstack/blob/master/apps/rush/CHANGELOG.md) to demonstrate the level of detail expected.
-
-## Updating dependencies/devDependencies on packages within the monorepo
-
-The version numbers of internal dependencies should not be manually edited.
-These will be automatically updated by the overall *version bump* workflow.
-Note that the packages are published by CI builds only.
-
-## Updating dependencies/devDependencies on packages external to monorepo
-
-Use these instructions to update dependencies and devDependencies on external packages (ones that live outside of this monorepo).
-
-1. Edit the appropriate `package.json` file to update the semantic version range
-2. Run `rush check` to make sure that you are specifying consistent versions across the repository
-3. Run `rush update` to make sure the newer version of the module specified in #1 is installed
-
-## Other NPM Scripts
-
-1. Build TypeDoc documentation for all packages: `rush docs`
-2. Build TypeDoc documentation for a single package: `cd core\backend` and then `rushx docs`
 
 ## Contribution
 
```

Comment:
I also think we should be very explicit in what can be found in the contributing guide. We  should mention that thats where workflows, contribution standards, and FAQ can be found.

Reply:
okay, I've added some more info to this section.